src/app/backend/resource/cronjob/jobs.go

```diff
@@ -144,3 +145,41 @@ func filterJobsByState(active bool, jobs []batch.Job) (matchingJobs []batch.Job)
 	}
 	return
 }
+
+// TriggerCronJob manually triggers a cron job and creates a new job.
+func TriggerCronJob(client client.Interface,
+  namespace, name string) error {
+
+  cronJob, err := client.BatchV1beta1().CronJobs(namespace).Get(name, metaV1.GetOptions{})
+
+  if err != nil {
+    return err
+  }
+
+  annotations := make(map[string]string)
+  annotations["cronjob.kubernetes.io/instantiate"] = "manual"
+
+  labels := make(map[string]string)
+  for k, v := range cronJob.Spec.JobTemplate.Labels {
+    labels[k] = v
+  }
+
+  jobToCreate := &batch.Job{
+    ObjectMeta: metaV1.ObjectMeta{
+      // job name cannot exceed DNS1053LabelMaxLength (52 characters)
```

Comment:
CronJob name can be long already though, and especially in a UI, we should perhaps help the enduser with that.